jmxtrans-output/jmxtrans-output-cloudwatch/src/test/java/com/googlecode/jmxtrans/model/output/MapEntryToDimensionTest.java

```diff
@@ -64,14 +83,25 @@ public void valueMustBeGiven() {
 	}
 
 	@Test
-	@Ignore("Should run on EC2 to be actually relevant")
+	//@Ignore("Should run on EC2 to be actually relevant")
```

Comment:
Will it run on Travis or any computer?
Maybe use an assumption (assumeXxx(..)) to check test is ran on AWS?